doc/API/centreon-api-v2.yaml

```diff
@@ -466,6 +466,8 @@ paths:
 
         * id
         * name
+        * alias
+        * fqdn
```

Comment:
it seems not possible to search by fqdn (only by address)
Could you add this possibility ?